### ISSUE TITLE
quicklisp overrides?

### ISSUE THREAD

**Issue by atgreen** [container-lisp/s2i-lisp#9]
After much messing around, the quicklisp-mirror container is working perfectly.  I actually found a bug in hunchentoot that the devs fixed, but now quicklisp-mirror has to carry its own hunchentoot (and usocket, because of an API change) until the next quicklisp release.  This required some annoying hacks in the quicklisp-mirror assemble script, but I think we can make it easier for people.   One idea is, if your source code contains a local-projects dir, copy that into the quicklisp local-projects directory before running the build.  Does this make sense to you?

**Comment by hjudt** [comment]
Yes, it does make sense. I have already thought about the possibility of including local projects in the past, but was unsure what would be the best way to do it. Is including a local-projects directory in the source code repository the best way to do it, or can you think of alternatives that we should perhaps take into account?
